app/controllers/TaxFreeAmountController.scala

```diff
@@ -46,7 +48,7 @@ class TaxFreeAmountController @Inject()(
   taxFreeAmount: TaxFreeAmountView,
   implicit val templateRenderer: TemplateRenderer,
   errorPagesHandler: ErrorPagesHandler)(implicit ec: ExecutionContext)
-    extends TaiBaseController(mcc) {
+    extends TaiBaseController(mcc) with Logging {
```

Comment:
How come you need `with Logging`, out of interest? - I thought it would be available from import

Reply:
It's a different type of logger I believe, the same that is extended in the TaxAccount Connector so keeping the same functionality after I removed the Exception case in the connector